pwndbg/heap/ptmalloc.py

```diff
@@ -393,6 +393,9 @@ def __init__(self, addr, arena=None) -> None:
                 self._gdbValue = None
 
         self.start = self._memory_region.start
+        # i686 alignment heuristic
```

Comment:
Any reason to prefer this over `if arch.current == 'i386'`?

Reply:
In my rewrite (that was reverted), I think I went with explicitly checking if the arch is i386. Otherwise this if-statement may hide bugs on other architectures.

I don't feel too strongly about this, will go with whatever you think is best.

Reply:
The original check was written by @matrix1001 and reviewed by @GrosQuildu in 2019.
I believe the consensus was that this heap realignment may occur when there's a discrepancy between compiled & runtime alignment requirements e.g. 32bit binaries running on a 64bit system. So perhaps not unique to i386 🤷